examples/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file? Looks like resources has pomless enabled.

Reply:
As far as I understand, the pomless aggregator only works for a [fixed](https://github.com/eclipse/tycho/blob/master/tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/TychoAggregatorMapping.java) set of directories.

Both the _bundles_ and _tests_ directory are within this set but _examples_ is not. Which is why the pom file is not necessary for those cases.

I've also tried specifying the artifact directly. (i.e. via \<module\>example/org.eclipse.ui.examples.filesystem\</module\>)
But in this case Tycho still requires a pom file in the _examples_ directory. Here's the error:

> Non-readable POM C:\[...]\eclipse.platform.resources\examples\org.eclipse.ui.examples.filesystem\.polyglot.META-INF: No parent pom file found in C:\[...]\eclipse.platform.resources\examples